app/Comment.php

```diff
@@ -35,13 +37,39 @@ public function getPostId()
         return $this->attributes['post_id'];
     }
 
-    public function setPostId($pId)
+    public function setPostId($user_id)
+    {
+        $this->attributes['user_id'] = $user_id;
+    }
```

Comment:
This method is not clear at all. Why do you use post and the user?

Reply:
It is a mistake.